setup.py

```diff
@@ -23,8 +24,9 @@
 ]
 
 REQUIREMENTS = [
-    'Django>=1.8,<2.0',
-    'django-mptt>=0.4.2,<0.9.2',
+    'six',
+    'Django~=1.11.0',
+    'django-mptt~=0.9.0',
```

Comment:
Another trick is to read and include the contents of requirements files here to avoid duplication.